esmvaltool/utils/cmorizers/obs/utilities.py

```diff
@@ -126,26 +126,30 @@ def read_cmor_config(cmor_config):
 def save_variable(cube, var, outdir, attrs, **kwargs):
     """Saver function."""
     # CMOR standard
-    cube_time = cube.coord('time')
-    reftime = Unit(cube_time.units.origin, cube_time.units.calendar)
-    dates = reftime.num2date(cube_time.points[[0, -1]])
-    if len(cube_time.points) == 1:
-        year = str(dates[0].year)
-        time_suffix = '-'.join([year + '01', year + '12'])
-    else:
-        date1 = str(dates[0].year) + '%02d' % dates[0].month
-        date2 = str(dates[1].year) + '%02d' % dates[1].month
-        time_suffix = '-'.join([date1, date2])
-
-    file_name = '_'.join([
+    try:
+        cube_time = cube.coord('time')
+        reftime = Unit(cube_time.units.origin, cube_time.units.calendar)
```

Comment:
It would be a bit nicer to put the code from line 131 till line 139 in an `else` clause, that way it is immediately clear where the exception is expected to be raised, i.e. on line 130.

Reply:
I didn´t know this was possible, thanks. Done!